spyderlib/plugins/configdialog.py

```diff
@@ -731,6 +731,12 @@ def setup_page(self):
                                            'windows_style',
                                            default=self.main.default_style)
 
+        themes = ['Spyder 2', 'Spyder 3']
+        icon_choices = list(zip(themes, [theme.lower() for theme in themes]))
+        icons_combo = self.create_combobox(_('Icon theme'), icon_choices,
+                                           'icon_theme', default='spyder 2',

```

Comment:
And please remove the `default` value from here, so it's only grabbed from `config.py` and we can update it without problems in the future.